### PR TITLE
Bug 1810997: pkg/operator/etcdcertsigner: always include ::1 in cert SAN

### PR DESCRIPTION
**Pull request by hexfusion**
We should not worry about conditionally doing this always add.

### CODE REVIEW

File: pkg/operator/etcdcertsigner/etcdcertsignercontroller.go [hunk 1]
```diff
@@ -305,6 +305,7 @@ func (c *EtcdCertSignerController) createSecretForNode(node *corev1.Node) error
 		"etcd.openshift-etcd.svc.cluster.local",
 		"*." + etcdDiscoveryDomain,
 		"127.0.0.1",
+		"::1",
```
Comment (deads2k) [inline pkg/operator/etcdcertsigner/etcdcertsignercontroller.go]: in theory, this is the same as below.

**Comment by deads2k** [comment]
eh, why not.  How much worse can it get

/lgtm

File: pkg/operator/etcdcertsigner/etcdcertsignercontroller.go (hunk 1 above)
Comment (hexfusion) [reply pkg/operator/etcdcertsigner/etcdcertsignercontroller.go]: Yeah handshake does not seem smart enough to understand long form.

**Pull request merged**